Tests/BackendIntegrationTests/MainThreadMonitor.swift

```diff
@@ -29,6 +29,11 @@ final class MainThreadMonitor {
     }
 
     func run() {
+        guard !Self.debuggerIsAttached else {
+            Logger.verbose("\(type(of: self)): debugger is attached, ignoring")
+            return
+        }
```

Comment:
So when you run tests through Xcode, the debugger will be attached by default I think. I guess that means this monitor will only run when executing tests through the command line tool correct? We should just be aware that tests results may vary between executing them through xcode and through command line.

Reply:
Yeah or in CI.